util/metrics/reporter.go

```diff
@@ -205,10 +206,11 @@ func (reporter *MetricReporter) tryInvokeNodeExporter(ctx context.Context) {
 			os.Stderr}
 	}
 	// prepare the vargs that the new process is going to have.
-	vargs := []string{
-		reporter.serviceConfig.NodeExporterPath,
-		"--web.listen-address=" + reporter.serviceConfig.NodeExporterListenAddress,
-		"--web.telemetry-path=" + nodeExporterMetricsPath}
+	whitespaceRegExp := regexp.MustCompile(`\s+`)
+	vargs := whitespaceRegExp.Split(reporter.serviceConfig.NodeExporterPath, -1)
```

Comment:
instead of splitting on whitespace, it'd be safer to add `NodeExporterArgs []string` to ServiceConfig struct

Reply:
I was hesitant to do this as I thought it might require bumping the config version. There are a lot of warnings in that file so I thought it best to leave as is.

Reply:
Thanks for being cautious.  The config version applies to the Local config struct version.

Reply:
Thanks David I realized that after I commented.